students/hyska_monika/lesson_11_argparse_math/regex_search.py

```diff
@@ -0,0 +1,60 @@
+"""
+Program that opens all .txt files in a folder and
+searches for any line that matches a user-supplied regular expression.
+The results should be printed to the screen.
+"""
+
+import re
+import os
+import argparse
+import sys
+
+
+def check_arg(args=None):
```

Comment:
`read_args` would be the right name. `check` suggests at most raising an exception if something goes wrong, it doesn't suggest any return value.